pulp_ansible/app/logutils.py

```diff
@@ -19,7 +17,8 @@ def emit(self, record):
         # This import cannot occur at import time because Django attempts to instantiate it early
         # which causes an unavoidable circular import as long as this needs to import any model
         from .models import CollectionImport
+        from pulpcore.plugin.models import Task
```

Comment:
Is this code available as far back as `3.12.2` ? https://github.com/pulp/pulp_ansible/blob/master/requirements.txt#L5

Reply:
not sure, I have to check

Reply:
yes, it is there: https://github.com/pulp/pulpcore/blob/3.12/pulpcore/app/models/task.py#L293-L304